lexnlp/extract/batch/__init__.py

```
@@ -29,17 +29,33 @@
 
 from lexnlp.extract.batch.async_extract import (
```

Pylint (4.0.5)

[error] Cannot import 'lexnlp.extract.batch.async_extract' due to 'invalid syntax (lexnlp.extract.batch.async_extract, line 72)'

(E0001)